code/game/gamemodes/shadowling/ascendant_shadowling.dm

```diff
@@ -25,9 +28,16 @@
 
 	minbodytemp = 0
 	maxbodytemp = INFINITY
-	environment_smash = 2
+	environment_smash = 3
 
 	faction = list("faithless")
 
 /mob/living/simple_animal/ascendant_shadowling/Process_Spacemove(var/movement_dir = 0)
 	return 1 //copypasta from carp code
+
+/mob/living/simple_animal/ascendant_shadowling/say(var/message)
+	message = message + "!!"
```

Comment:
this should be an override of treat_message(), you should basically never override say() directly.